src/docs/GetStarted/Developers/index.mdx

```diff
@@ -3,30 +3,23 @@ title: 'Developer guide'
 order: 3
 ---
 
-**[Installation](Installation/)**<br />
-Where to find the dependencies, and the essentials requirements
-<br /><br />
+**[Installation](Installation/)**  
+Where to find the dependencies, and the essential requirements
 
-**[what's included ?](whatsincluded/)**<br />
+**[what's included ?](whatsincluded/)**  
```

Comment:
~~what's included~~ => What's inside

Reply:
why ?

Reply:
Sorry I made a mistake, I meant to say : 
~~what's included~~ => What's included

Please put the first letter in uppercase as the others subtitles.

Reply:
ok this will be replaced with Remy's work either way, so if it is the only comment, I think we can merge It as is.